README.md

```diff
@@ -12,6 +12,9 @@ with:
 - Digital signatures
 - Cryptographic hashing
 
+## Blockchain main goal
+Watch [this video](https://www.youtube.com/watch?v=RplnSVTzvnU) (Understanding Blockchain function). (Est. time: 15 minutes)
```

Comment:
Please refer to other comments on this TED Talk video feedback.

Reply:
Could you give me more details ? I don't understand what you said.

Reply:
Just making reference to my previous comment. No worries.